dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/support/FailbackRegistryTest.java

```diff
@@ -180,6 +181,27 @@ public void notify(List<URL> urls) {
         assertEquals(2, count.get());
     }
 
+    @Test
+    public void testRecover() throws Exception {
+        CountDownLatch countDownLatch = new CountDownLatch(2);
+        final AtomicReference<Boolean> notified = new AtomicReference<Boolean>(false);
+
+        NotifyListener listner = new NotifyListener() {
+            @Override
+            public void notify(List<URL> urls) {
+                notified.set(Boolean.TRUE);
+            }
+        };
+
+        MockRegistry mockRegistry = new MockRegistry(registryUrl, countDownLatch);
+        mockRegistry.register(serviceUrl);
+        mockRegistry.subscribe(serviceUrl, listner);
+        Assert.assertEquals(mockRegistry.getRegistered().size(), 1);
+        Assert.assertEquals(mockRegistry.getSubscribed().size(), 1);
+        mockRegistry.recover();
+        Assert.assertEquals(mockRegistry.getFailedSubscribed().size(), 1);
+        Assert.assertEquals(countDownLatch.getCount(), 0);

```

Comment:
The default is to retry every 5 seconds，relying on this value of `0` will cause this test case to be unstable?